Add data

kaminow opened the pull request:
Add all the data files and update the README file.

kaminow committed:
Add first data file.

kaminow committed:
Update README.

kaminow committed:
Add unfiltered data file.

kaminow committed:
Update README.

jchodera commented on README.md:
```diff
@@ -1 +1,23 @@
+## Files
+* `cdd_noncovalent_dates_2023_10_18_unfiltered_clean.csv`: All noncovalent available
+data as of October 10, 2023, cleaned to remove data without a noted SMILES string or
+IC50 value
```
Does this mean numerical IC50, or does this include measurements below or above detectable range?

What was the original source of the data before cleaning? CDD Vault?

Do you mean "filtered" instead of "cleaned"?

Is the data indexed by suspected_SMILES?

Have we documented what all the fields mean?

Did we note this was from the fluorescence assay and cite the experimental assay description in the preprint?

jchodera replied:
Can we caution that this will contain some racemates?

kaminow replied:
1. these are all semi-quantitative data (I can make a note of that in the README). I did include the `In_range` field that lets the user know if a given measurement was below, above, or within the assay range
2. this is all from the CDD vault. I can make a note of that as well
3. for this dataset, I've removed all data that doesn't have a SMILES or IC50 value, but have done no other filtering. to me, that's "cleaning", but if "cleaning" typically implies that no data points were removed, I can change the wording to "filtered"
4. what do you mean by indexed? the first column for both CSV files is the PostEra ID, but I can rearrange the files so SMILES is first
5. yes, the next section in the README explains the column headers
6. I have not, but I'll include that in the README

jchodera replied:
* Can you modify the README to clarify these points?
* I'd use "filtering", since "cleaning" to me implies changing data rather than just omitting some data.
* Re "indexed": The first column in the file should be the SMILES we want people to use (suspected_SMILES), and the second column should be the title (PostEra ID). This is how OpenEye and other readers expect the data to be arranged.

JenkeScheen commented on README.md:
```diff
@@ -1 +1,23 @@
+## Column Headers
+A brief explanation of each column present in the CSV files.
+* `Canonical_PostEra_ID`: Molecule's canonical name
+* `suspected_SMILES`: SMILES string of the synthesized molecule
```
is there somewhere we can refer to for a clarification on this? I imagine externals might not immediately understand the difference between 'regular' and 'suspected' SMILES.

kaminow replied:
would it be better if I just renamed the field to `SMILES`? or would the be obfuscating too much?

JenkeScheen replied:
you can just post this in the readme, copied from https://github.com/postera-ai/COVID_moonshot_submissions#details-on-smiles-and-molecule-identities:

On `suspected_SMILES`: Note that many of the enantiopure compounds on this project were obtained by chiral separation, and thus the compounds are often obtained as a single enantiomer with unknown absolute stereochemistry. The `suspected_SMILES` column represents the understanding of the actual identity of the compound given the current information, which explains the frequent use of enhanced/relative stereochemistry representations. Keep in mind that the `suspected_SMILES` is often not the same as the `registration_SMILES`, since compounds must be registered as a real (non-relatively defined) compound on this project.

kaminow replied:
sounds good, done in https://github.com/asapdiscovery/COVID_moonshot_data_clean/pull/1/commits/aabc260f0485cb95d2a957f4faa91e1651a9145f

jchodera replied:
In addition to adding this note to the README, I would keep `suspected_SMILES` as the first column, PostEra ID in the second column, and `registration_SMILES` as another column.

kaminow committed:
Clarification on semi-quantitative data.

kaminow committed:
More data clarifications.

kaminow committed:
Add preprint citation for IC50 fluorescence measurement.

jchodera commented on README.md:
```diff
@@ -1 +1,23 @@
+in µM units
+* `IC50_CI_(Upper)_(µM)`: Upper bound of the 95% CI of the experimental IC50 value,
+in µM units
+* `Hill_slope`: Calculated slope of the Hill equation for this measurement
```
Can you change `Curve_class` to `CDD_curve_class` and document that it is the CDD curve class?

jchodera commented on cdd_achiral_enantiopure_dates_2023_10_18_filt.csv:
```diff
@@ -0,0 +1,804 @@
+Canonical_PostEra_ID,suspected_SMILES,IC50_(µM),IC50_CI_(Lower)_(µM),IC50_CI_(Upper)_(µM),Hill_slope,Curve_class,Semiquant
```
This file will need to have its columns permuted so that `suspected_SMILES,Canonical_PostEra_ID` appear in that order, and the `registration_SMILES` appears as well.

jchodera commented on cdd_noncovalent_dates_2023_10_18_unfiltered_clean.csv:
```diff
@@ -0,0 +1,2147 @@
+Canonical_PostEra_ID,suspected_SMILES,IC50_(µM),IC50_CI_(Lower)_(µM),IC50_CI_(Upper)_(µM),Hill_slope,Curve_class,Semiquant
```
This file will need to have its columns permuted so that `suspected_SMILES,Canonical_PostEra_ID` appear in that order, and the `registration_SMILES` appears as well.

jchodera reviewed (changes requested):
Can we include the dates in the file as well, so folks can do temporal splits?

kaminow committed:
Swap PostEra ID and SMILES columns.

kaminow committed:
Rename cleaned -> filtered and update README accordingly.

kaminow committed:
Rename Curve_class -> CDD_curve_class and update README.

kaminow commented:
I think I've addressed all your comments @jchodera. some followup questions on my end:
1. there doesn't seem to be any `registration_SMILES` field in CDD or in the dataset in https://github.com/asapdiscovery/COVID_moonshot_submissions. is this the same as `shipment_SMILES` or is there something missing?
2. some of the SMILES are CXSMILES, should I strip out the extra information (converting them to just standard SMILES), or leave in the extra info?
3. some of the SMILES have disconnected compounds (ie there are `.` in the SMILES string), should I leave this as containing all the compounds, or just keep the largest (or some other method)?

jchodera commented on README.md:
```diff
@@ -1 +1,45 @@
+> The COVID Moonshot Consortium, John Chodera, Alpha Lee, Nir London, and Frank von
+Delft. “Open Science Discovery of Oral Non-Covalent SARS-COV-2 Main Protease
+Inhibitors.” ChemRxiv, October 19, 2021.
+https://doi.org/10.26434/chemrxiv-2021-585ks-v2.
```
Please use the bioRxiv manuscript. The chemrxiv one has not been updated.

jchodera commented on README.md:
```diff
@@ -1 +1,45 @@
+On `suspected_SMILES`: Note that many of the enantiopure compounds on this project were
+obtained by chiral separation, and thus the compounds are often obtained as a single
+enantiomer with unknown absolute stereochemistry. The `suspected_SMILES` column
+represents the understanding of the actual identity of the compound given the current
```
Current unceratanding from analysis of the X-ray structures of related compounds.

jchodera reviewed (approved):
Two quick requests for changes and then we can merge!

JenkeScheen committed:
last readme changes
